src/vmm/src/utils/mod.rs

```diff
@@ -13,6 +13,9 @@ pub mod sm;
 use std::num::Wrapping;
 use std::result::Result;
 
+/// How many bits to left-shift by to convert KiB to bytes.:
+pub const KIB_TO_BYTES_SHIFT: usize = 20;
```

Comment:
hmm 20 is to shift from bytes to MiB (2^20), or viceversa. KiB (2^10) should be 10 no? Should the constant be `MIB_TO_BYTES_SHIFT`?

Reply:
you are absolutely correct!

Reply:
fixed! this is the second time something like this has happened to me, maybe y'all should stop letting me touch code related to units :rofl: